src/tracer.js

```diff
@@ -1,20 +1,27 @@
 'use strict'
 
+const opentracing = require('opentracing')
 const Tracer = require('./opentracing/tracer')
 
+const noop = new opentracing.Span()
+
 class DatadogTracer extends Tracer {
   constructor (config) {
     super(config)
 
     let ScopeManager
+    let Scope
 
     if (process.env.DD_CONTEXT_PROPAGATION === 'false') {
       ScopeManager = require('./scope/noop/scope_manager')
+      Scope = require('./scope/new/base')
     } else {
       ScopeManager = require('./scope/scope_manager')
```

Comment:
This is also just a `noop/scope_manager` right?

Reply:
if so we can just do `let ScopeManager = require('./scope/noop/scope_manager');`

Reply:
No because it has a completely different API.